R/utils.R

```diff
@@ -1,3 +1,26 @@
+# Copied from shiny 0.11.1.9003
+toJSON <- function(
+  x, ...,  dataframe = "columns", null = "null", na = "null", auto_unbox = TRUE,
+  digits = getOption("shiny.json.digits", 16), use_signif = TRUE, force = TRUE,
+  POSIXt = "ISO8601", UTC = TRUE, rownames = FALSE, keep_vec_names = TRUE
+) {
+  jsonlite::toJSON(
+    I(x), dataframe = dataframe, null = null, na = na, auto_unbox = auto_unbox,
+    digits = digits, use_signif = use_signif, force = force, POSIXt = POSIXt,
+    UTC = UTC, rownames = rownames, keep_vec_names = keep_vec_names, ...
+  )
+}
+
+if (requireNamespace('shiny')) local({
```

Comment:
This is a note to ourselves (authors of this htmlwidgets package) at build time. It may be confusing to end users if they see this warning, since they cannot really do anything about it.